remote-monitoring/single-vm-setup.sh

```diff
@@ -9,6 +9,10 @@ UPDATE="${DEST}/update.sh"
 LOGS="${DEST}/logs.sh"
 SIMULATE="${DEST}/simulate.sh"
 
+CERTS="${DEST}/certs"
+CERT="${CERTS}/tls.crt"
+PKEY="${CERTS}/tls.key"
```

Comment:
folders and files should be read-only and accessible only to root (or the UID used by nginx) - 0550 should do

Reply:
Fixed it